sqlengine/postgres_engine.go

```diff
@@ -317,6 +317,22 @@ func (d *PostgresEngine) JDBCURI(address string, port int64, dbname string, user
 	return fmt.Sprintf("jdbc:postgresql://%s:%d/%s?%s", address, port, dbname, params.Encode())
 }
 
+const createExtensionPattern = `CREATE EXTENSION IF NOT EXISTS "{{.extension}}"`
+
+func (d *PostgresEngine) CreateExtensions(extensions []string) error {
+	for _, extension := range extensions {
+		createExtensionTemplate := template.Must(template.New(extension + "Extension").Parse(createExtensionPattern))
```

Comment:
The documentation is confusing, because [`QueryRow`](https://golang.org/pkg/database/sql/#DB.QueryRow) for example uses a `?` as a placeholder. Can't seem to find the relevant documentation on its proper usage.